File: Python 3/10195 UVA.py
import math

#sys.stdin = open("in.txt","r")
#sys.stdout = open("out.txt","w")

while True:
    try:
        a,b,c = map(float,input().split())
        s = (a+b+c) / 2.0
        if s == 0.0:
            print("The radius of the round table is: 0.000")
        else:
            area = math.sqrt(s*(s-a)*(s-b)*(s-c))
            r = area / s
            print("The radius of the round table is: {0:0.3f}".format(r))
    except:
        break

File: Java/10195 UVA.java
import java.util.*;

public class Main
{
    public static void main(String[] args)
    {
        Scanner sc = new Scanner(System.in);
        while (sc.hasNextDouble())
        {
            double a = sc.nextDouble();
            double b = sc.nextDouble();
            double c = sc.nextDouble();

            double s = (a+b+c)*0.5;

            if(s==0.0000)
                System.out.printf("The radius of the round table is: 0.000\n");

            else
            {
                double area = Math.sqrt(s*(s-a)*(s-b)*(s-c));
                double r = area / s;
                System.out.printf("The radius of the round table is: %.3f\n",r);
            }
        }
        System.exit(0);
    }
}
